src/main/java/application/TodoResource.java

```diff
@@ -30,4 +32,12 @@ public Response getTodos() {
     return Response.ok().entity(list).build();
   }
 
+  @GET
+  @Path("/{todoNo}")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response getTodoById(@PathParam("todoNo") int todoNo){
+    List<Todo> list = todoService.listTodo();
+    return Response.ok().entity(list.get(todoNo)).build();
```

Comment:
Anstatt das hier zu machen, erstelle dir bitte eine entsprechende Methode im Service, der dort schaut _ob_ es das To-Do überhaupt gibt und eins zurückgibt. Wenn nicht, sollte diese Methode hier nicht "OK" zurückgeben, sondern "NOT_FOUND".

Logik gehört nach "domain", also in deinen Service.

Reply:
Es wäre gut, wenn du nicht nur die Methode sondern auch die Parameter umbenennst. Auch in Zeile 36.